Framework/API/src/MultipleFileProperty.cpp

```diff
@@ -395,6 +414,10 @@ MultipleFileProperty::setValueAsMultipleFiles(const std::string &propValue) {
     allFullFileNames.push_back(fullFileNames);
   }
 
+  // cache the new version of things
+  m_oldPropValue = propValue;
+  m_oldFoundValue = allFullFileNames;
+
   // Now re-set the value using the full paths found.
   return PropertyWithValue<std::vector<std::vector<std::string>>>::setValue(
       toString(allFullFileNames));
```

Comment:
Would it make more sense to use `PropertyWithValue<TYPE>::operator=(const TYPE &)` and avoid additional conversions between strings and vectors?